target/linux/qualcommax/image/ipq60xx.mk

```
@@ -86,6 +86,24 @@ define Device/tplink_eap610-outdoor
 endef
 TARGET_DEVICES += tplink_eap610-outdoor
 
+define Device/tplink_eap620hd-v2
+        $(call Device/FitImage)
+        $(call Device/UbiFit)
+        DEVICE_VENDOR := TP-Link
+        DEVICE_MODEL := EAP620HD-V2
```

Please separate `V2` into `DEVICE_VARIANT`.

```suggestion
        DEVICE_MODEL := EAP620HD
        DEVICE_VARIANT := v2
```